lib/bamboo/adapters/mailgun_helper.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Bamboo.MailgunHelper do
+  @moduledoc """
+  Functions for using features specific to Mailgun (e.g. tagging, templates).
+  """
+
+  alias Bamboo.Email
+
+  @doc """
+  Add a tag to outgoing email to help categorize traffic based on some criteria, perhaps separate signup emails from password recovery emails or from user comments.
+
+  More details can be found in the [Mailgun documentation](https://documentation.mailgun.com/en/latest/user_manual.html#tagging)
+
+  ## Example
+
+      email
+      |> MailgunHelper.tag("welcome-email")
+  """
+  # If/when https://github.com/thoughtbot/bamboo/issues/469 is tackled, this can be extended
+  # to support multiple tags. If you make that change, note in the above doc that the API only supports 3 tags
```

Comment:
Thanks for this comment! Our preference is for comments like this to live in GitHub, vs. directly in the code. Maybe add this comment directly on this PR and then link to it from the issue you referenced? Or file a new issue for it?